tests/daemon/test-persistent-cache.c

```diff
@@ -954,33 +953,20 @@ static void
 test_persistent_cache_does_not_compute_offset_when_boot_id_is_same (gboolean     *unused,
                                                                     gconstpointer dontuseme)
 {
-  EmerPersistentCache *cache = make_testing_cache ();
-
-  GError *error = NULL;
-  gboolean get_succeeded =
-    emer_persistent_cache_get_boot_time_offset (cache, NULL, &error);
-  g_assert_no_error (error);
-  g_assert_true (get_succeeded);
-  g_assert_true (boot_offset_was_reset ());
-
   gint64 relative_time;
   g_assert_true (emtr_util_get_current_time (CLOCK_BOOTTIME, &relative_time));
-
   gint64 absolute_time;
   g_assert_true (emtr_util_get_current_time (CLOCK_REALTIME, &absolute_time));
 
+  EmerPersistentCache *cache = make_testing_cache ();
+
+  g_assert_true (boot_offset_was_reset ());
+
   g_object_unref (cache);
   set_boot_id_in_metadata_file (FAKE_BOOT_ID);
 
   EmerPersistentCache *cache2 = make_testing_cache ();
 
-  // This call should have to compute the boot offset itself.
```

Comment:
This comment can be moved to describe the second `make_testing_cache` call rather than be deleted.